types/meteor/mongo.d.ts

```diff
@@ -219,6 +241,8 @@ declare module "meteor/mongo" {
             map<M>(callback: (doc: U, index: number, cursor: Cursor<T, U>) => M, thisArg?: any): Array<M>;
             observe(callbacks: ObserveCallbacks<U>): Meteor.LiveQueryHandle;
             observeChanges(callbacks: ObserveChangesCallbacks<T>, options?: { nonMutatingCallbacks?: boolean }): Meteor.LiveQueryHandle;
+            _mongo: MongoInternals.Connection;
+            _synchronousCursor: Mongo.Cursor<Mongo.Document>;
```

Comment:
Since we agreed that these are not needed, I'd remove it completely, along with the code added above.